fabulaws/library/wsgiautoscale/api.py

```diff
@@ -1280,21 +1280,20 @@ def mount_encrypted(drive_letter='f'):
     answer_sudo('cryptsetup luksOpen {device} {crypt}'
                 ''.format(device=device, crypt=crypt),
                 answers=answers)
-    sudo('sudo mount /dev/mapper/{0} /secure'.format(crypt))
     current_server = _current_server()
+    if exists(current_server.mount_script):
+        sudo(current_server.mount_script)
+    else:
+        abort('attempting to mount encrypted partitions, but mount script {} '
+              'does not exist'.format(current_server.mount_script))
     if exists(current_server.default_swap_file):
         sudo('swapon %s' % current_server.default_swap_file)
-    sudo('sudo mount -o bind /secure/home/secure /home/secure' % env)
```

Comment:
They are being moved to the `mount-deferred.sh` file. This allows the instance *creation* to determine what the mount commands should be rather than relying on this code being in sync with whatever happened to be 'current' as of the time the servers were created.